ccdproc/tests/test_image_collection.py

```diff
@@ -217,6 +217,17 @@ def test_generator_data(self, triage_setup):
         for img in collection.data():
             assert isinstance(img, np.ndarray)
 
+    def test_generator_ccddata(self, triage_setup):
+        from ccdproc import CCDData
+        collection = image_collection.ImageFileCollection(location=triage_setup.test_dir,
+                                             keywords=['imagetyp'])
+        for img in collection.ccddata():
+            assert isinstance(img, CCDData)
+
+        ccd_kwargs = {'unit': 'adu'}
+        for img in collection.ccddata(ccd_kwargs=ccd_kwargs):
```

Comment:
That's a really nitpicky comment but generally `CCDData` objects are labelled `ccd`, not `img`. 

Feel free to ignore this :smile: